### PR TITLE
empty flippers tip

### LINKED ISSUE #5638: Tip for empty flippers
Looks like it's quiet in here!

something like this? @daiagi 
- formatting same as in explore empty state
image

guess you need one for holders as well no? Or there is always the mint null address right

here is one for it anyway: Empty holder tab? Start collecting NFTs and make your mark at the top!

_Originally posted by @exezbcz in https://github.com/kodadot/nft-gallery/issues/5522#issuecomment-1497707041_

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5638
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -933,6 +933,7 @@
       "time": "Time"
     },
     "noResults": "Looks Like There's Nothing Here Yet",
+    "noFlips": " Looks like it's quiet in here!",
```
Comment (roiLeo) [inline locales/en.json:936]: I would remove exclamation mark
```suggestion
    "noFlips": "Looks like it's quiet in here",
```

File: components/collection/activity/ownerInsightsTabs/FlipperTab.vue [hunk 2]
```diff
@@ -1,73 +1,92 @@
...
-          <div>
-            <div
-              class="is-size-7 has-text-k-blue is-clickable"
-              @click="toggleNFTDetails(flipperId)">
-              {{ $t('activity.nftDetails') }}
-              <NeoIcon
-                :icon="
-                  isNFTDetailsOpen[flipperId] ? 'chevron-down' : 'chevron-right'
-                " />
+    <div v-if="flippers.length > 0">
```
Comment (roiLeo) [inline components/collection/activity/ownerInsightsTabs/FlipperTab.vue:3]:
```suggestion
    <div v-if="flippers.length">
```

File: components/collection/activity/ownerInsightsTabs/FlipperTab.vue [hunk 3]
```diff
@@ -1,73 +1,92 @@
...
-          <NFTsDetaislDropdown :flips="flips" variant="Flippers" />
-        </div>
+        <hr class="my-3 mx-5" />
+      </div>
+      <div ref="target" />
+    </div>
+
+    <div
+      v-else
+      class="is-flex is-justify-content-center is-align-items-center pt-6 px-4">
```
Comment (roiLeo) [inline components/collection/activity/ownerInsightsTabs/FlipperTab.vue:85]:
```suggestion
      class="is-flex is-justify-content-center is-align-items-center pt-6">
```

**Review by roiLeo — APPROVED** [review]
✅ wfm

**Comment by yangwao** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 36.7 USD/KSM ~ 0.545 $KSM
🧗 [EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L](https://kusama.subscan.io/account/EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)
🔗 [0x1d483464632aadeb3c00bc5a5942e0d58f8c2385baa44d49660e40a3eeec5ddb](https://kusama.subscan.io/extrinsic/0x1d483464632aadeb3c00bc5a5942e0d58f8c2385baa44d49660e40a3eeec5ddb)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**